backup_and_restore/application_backup_and_restore/installing/installing-oadp-ibm-cloud.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-ibm-cloud"]
+= Configuring the OpenShift API for Data Protection with {ibm-cloud-title}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-ibm-cloud
+:installing-oadp-ibm-cloud:
+:credentials: cloud-credentials
+
+
+toc::[]
+
+You install the {oadp-first} Operator on an {ibm-cloud-title} cluster to back up and restore applications on the cluster. You configure {ibm-cloud-object-storage} to store the backups.
+
+// configuring the IBM COS instance
+include::modules/configuring-ibm-cos.adoc[leveloffset=+1]
+// include the module for creating default secret
+include::modules/oadp-creating-default-secret.adoc[leveloffset=+1]
+// include the module for creating custom secret
+include::modules/oadp-secrets-for-different-credentials.adoc[leveloffset=+1]
```

Comment:
I would change order here. This section has the first DPA example, which does not have all required fields present. **Installing the Data Protection Application 1.3** section has a complete DPA example, would move it to be the first example DPA to be shown in the docs

Reply:
checking other cloud providers.. it does seem that other providers have the full dpa immediately following the secret setup.   I suspect since we're dropping the versioned docs for 1.1. / 1.2  we will find the 1.3 section in this doc is dropped and it will read more smoothly.   If that's the case, I'm find w/ the way it is as it wouldn't make sense to have the dpa posted twice.     Otherwise I agree w/ Mateus :)

Reply:
I agree with both of your comments. After the '1.2 removal' PR is merged, this will read better. But I do see the incomplete DPA in the custom secret module. Right now, I think it would not be a good idea to club that improvement in this PR. I say that because, getting through with reviews will take longer for this PR. So i will create a separate ticket and a PR to clean up the secret module so that we don't an incomplete DPA example. Thanks.